docs/ref/templates/builtins.txt

```diff
@@ -2430,18 +2430,16 @@ slightly different call:
     {% static "images/hi.jpg" as myphoto %}
     <img src="{{ myphoto }}"></img>
 
-.. note::
 
-    The :mod:`staticfiles<django.contrib.staticfiles>` contrib app also ships
-    with a :ttag:`static template tag<staticfiles-static>` which uses
-    ``staticfiles'`` :setting:`STATICFILES_STORAGE` to build the URL of the
-    given path (rather than simply using :func:`urllib.parse.urljoin` with the
-    :setting:`STATIC_URL` setting and the given path). Use that instead if you
-    have an advanced use case such as :ref:`using a cloud service to serve
-    static files<staticfiles-from-cdn>`::
+If :mod:`staticfiles<django.contrib.staticfiles>` contrib app is configured,
```

Comment:
"If the..."
I see you just move this, but you can use "~django.contrib.staticfiles" instead of "staticfiles<~django.contrib.staticfiles>"